spec/src/main/asciidoc/changelog.adoc

```diff
@@ -27,6 +27,7 @@ Changes marked with icon:bolt[role="red"] are breaking changes relative to previ
 ** Clarified that metric registry implementations are required to be thread-safe.
 
 * Changes in 2.0
+** Added ProcessCpuTime as a new optional base metric.
```

Comment:
I'd suggest a new 2.0.3 section even though it will look a bit weird that we don't have 2.0.1 and 2.0.2

Reply:
see my comment at https://github.com/eclipse/microprofile-metrics/pull/467#discussion_r335962128